ansible/www-standalone/resources/config/nodejs.org

```diff
@@ -57,6 +71,13 @@ server {
             add_header access-control-allow-origin *;
         }
     }
+
+    location = /404.html {
+        # We disable caching of 404 pages as we want to avoid CloudFlare accidentally thinking that a 404 is permanent
+        # many cases 404 pages happen due to the server reaching maximum concurrent file system open() requests
+        # disabling cache allows CloudFlare to re-evaluate the same $uri once our server recovers and then properly cache the page
```

Comment:
There's also the case where we publish a new release. Sometimes external automation tries to get the binaries before they are actually published to the server.

Reply:
Mind adding a suggestion block of what should be added? 👀